src/worker/jobs/rss/sync_crate_feed.rs

```diff
@@ -43,8 +47,13 @@ impl BackgroundJob for SyncCrateFeed {
 
         info!("Loading latest {NUM_ITEMS} version updates for `{name}` from the database…");
         let mut conn = ctx.deadpool.get().await?;
+        let crate_id = if let Some(crate_id) = self.crate_id {
+            crate_id
+        } else {
+            load_crate_id(name, &mut conn).await?
+        };
 
-        let version_updates = load_version_updates(name, &mut conn).await?;
+        let version_updates = load_version_updates(crate_id, &mut conn).await?;
```

Comment:
it's probably easiest to keep using `name` for now and migrate over to `crate_id` in the next PR when this PR here has been deployed to production. That way we won't need the `load_crate_id()` fn at all :)

Reply:
yeah, exactly. that should make it easier to migrate them.